agent/config/config.go

```diff
@@ -194,6 +197,10 @@ func get(args []string, l *logrus.Entry) (cfg *Config, configFileF string, err e
 		if cfg.Ports.Max == 0 {
 			cfg.Ports.Max = 51999
 		}
+		if cfg.WindowConnectedTime == 0 {
+			cfg.WindowConnectedTime = 24 * time.Hour
```

Comment:
@dliakhov I think that 24h makes the metric slow to respond without the risks of noise from false positives by using a high-sensitivity threshold. For example:

- agent has been running for _n_ days
- a change is made to PMM server
- agents lose connection to the server
- 10 minutes of lost connectivity is still >99%

I think that 1h is probably the best interval here, so that the metric is more responsive to both loss and recovery from loss.